### DISCUSSION TITLE
2.4.1 Release Schedule?

### DISCUSSION THREAD

**Discussion by DandyDeveloper** [goharbor/harbor#15954] [Q&A]
There's a few commits I'm waiting on. Do we have a release schedule for patch releases on Harbor?

**Comment by DandyDeveloper** [comment]
@wy65701436 Any news on the next patch release to bring 2.4 inline with master?

**Reply by wy65701436** [reply]
Usually, the patch release cadence is one month. But, if there is no issue/bug fix/CVE, at least critical, we could extend it.
